pkgs/applications/graphics/curtail/default.nix

```diff
@@ -0,0 +1,69 @@
+{ lib
+, python3
+, fetchFromGitHub
+, wrapGAppsHook
+, appstream-glib
+, desktop-file-utils
+, gettext
+, gtk3
+, meson
+, ninja
+, pkg-config
+, gobject-introspection
+, jpegoptim
+, libwebp
+, optipng
+, pngquant
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "curtail";
+  version = "1.3.0";
```

Comment:
```suggestion
  version = "1.3.0";
  format = "other";
```

Reply:
I don’t see why? The format declaration is already the first build-related instruction after the inputs.
Why remove the comment? Isn’t it helpful?